api/controllers/OpenEAppController.js

```diff
@@ -33,40 +33,43 @@ const OpenEAppController = {
                     req,
                     res
                 );
+            const [casebookData] = casebookResponse;
+            const casebookStatus = casebookData.status || 'Not available';
+            const casebookDocuments = casebookData.documents || [];
+
             const pageData = OpenEAppController._formatDataForPage(
                 applicationTableData,
-                casebookResponse[0]
+                casebookData
             );
             const userRef = await OpenEAppController._getUserRef(
-                casebookResponse[0],
+                casebookData,
                 res
             );
             const daysLeftToDownload =
-                casebookResponse[0].status === 'Completed'
+                casebookData.status === 'Completed'
                     ? OpenEAppController._calculateDaysLeftToDownload(
-                          casebookResponse[0],
+                          casebookData,
                           req
                       )
                     : 0;
             const applicationExpired =
                 OpenEAppController._hasApplicationExpired(
-                    casebookResponse[0],
+                    casebookData,
                     daysLeftToDownload
                 );
 
-            const noOfRejectedDocs = OpenEAppController._calculateRejectedDocs(
-                casebookResponse[0]
-            );
+            const noOfRejectedDocs =
+                OpenEAppController._calculateRejectedDocs(casebookData);
 
             res.view('eApostilles/openEApp.ejs', {
                 ...pageData,
                 userRef,
                 user_data: userData,
                 daysLeftToDownload,
                 applicationExpired,
-                applicationStatus: casebookResponse[0].status,
+                applicationStatus: casebookStatus,
                 allDocumentsRejected:
-                    noOfRejectedDocs == casebookResponse[0].documents.length,
+                    noOfRejectedDocs == casebookDocuments.length,
```

Comment:
May want to consider changing these to === to check the type as well - minor but good practice

Reply:
Good spot 👍